### ISSUE TITLE
WARNING:spacy:[W108] The rule-based lemmatizer did not find POS annotation for the token 'NM'.

### ISSUE THREAD

**Issue by sushilr007** [microsoft/presidio#692]
**Describe the bug**
I am running following code and getting lot of warnings from Spacy. which slows down the processing.
```
from presidio_analyzer import AnalyzerEngine
engine = AnalyzerEngine()
response = engine.analyze(correlation_id=0, text = data, entities=[], language='en', all_fields=True, score_threshold=0.5)

```    

`WARNING:spacy:[W108] The rule-based lemmatizer did not find POS annotation for the token 'NM'. Check that your pipeline includes components that assign token.pos, typically 'tagger'+'attribute_ruler' or 'morphologizer'.`

**Additional context**
*Versions of packages:*
```
presidio-analyzer==2.2.0
spacy==3.0.5
spacy-legacy==3.0.5
```

**Comment by omri374** [comment]
Hi @sushilr007, thanks for your issue!
We are aware of this, and a PR was already merged to fix it (#682)

It will be updated on PyPI on our next release, early next week.

**Comment by navalev** [comment]
Hi @sushilr007, we just releaeed a new version to pypi, this issue should be resolved. Closing the issue, please re-open if needed
